core/bin/dust_api.rs

```diff
@@ -143,6 +143,95 @@ async fn projects_create(
     }
 }
 
+/// Clones a project.
+/// Simply consists in cloning the latest dataset versions, as we don't copy runs and hence specs.
+
+async fn projects_clone(
+    extract::Extension(state): extract::Extension<Arc<APIState>>,
+    extract::Path(project_id): extract::Path<i64>,
+) -> (StatusCode, Json<APIResponse>) {
+    let cloned = project::Project::new_from_id(project_id);
```

Comment:
The naming here was confusing for me. `cloned` implies that this is the cloned project (i.e. the new one), not the one that is being cloned (the parent). Maybe use a name like "parent" or "project_to_be_cloned"?